zk/txpool/pool.go

```diff
@@ -1199,6 +1199,14 @@ func (p *TxPool) addLocked(mt *metaTx, announcements *types.Announcements) Disca
 			return NotReplaced
```

Comment:
leave an Info here, that tells that tx was attempted to be replaced

Reply:
done.